src/lib/services/project-health-service.ts

```diff
@@ -52,11 +56,13 @@ export default class ProjectHealthService {
             hoursToMilliseconds(1),
         ).unref();
         this.featureToggleService = featureToggleService;
+        this.favoritesService = favoritesService;
     }
 
     // TODO: duplicate from project-service.
     async getProjectOverview(
         projectId: string,
+        userId?: number,
```

Comment:
I'd add optional field at the end of a method signature so that the default param for archived can work in more situations